modules/rostests/apitests/shell32/shell32_apitest_sub.cpp

```diff
@@ -179,6 +179,8 @@ static BOOL InitSHCN(HWND hwnd)
         default:
         {
             assert(0);
+            sources = 0;
+            events = EVENTS;
```

Comment:
Alternatively, set these as the default values of the variables at lines 94, 95.